.github/workflows/ci.yml

```diff
@@ -159,6 +159,41 @@ jobs:
       with:
         action: ${{ matrix.test.action }}
 
+  semver-checks:
```

Comment:
I think this will break `ci-main.yml` and `ci-merge-queue.yml`, which both invoke `ci.yml`, but won't have `context.issue.number` or `github.event.pull_request.base.sha`. Did you mean for this to be added to `ci-pr.yml` and `ci-pr-forks.yml`?

Reply:
yes, didn't understand how CI works apparently